users-backend/users/domain/repository.py

```diff
@@ -42,8 +41,12 @@ class UsersRepository(AbstractUsersRepository):
     def __init__(self, session: Optional[AsyncSession] = None) -> None:
         self.session = session
 
-    async def get_users(self) -> Paginable[User]:
-        return Paginable(self.session, User)
+    async def get_users(self, paginate=True) -> Union[list[User], Paginable[User]]:
```

Comment:
can you maybe just implement the query in this PR so we can discuss this change when more in context?